README.md

```diff
@@ -41,128 +41,17 @@ You can find an overview over the features available in Alacritty [here](./docs/
 
 ## Installation
 
-Some operating systems already provide binaries for Alacritty, for everyone
-else the instructions to build Alacritty from source can be found [here](INSTALL.md).
+Alacritty can be installed by using various package managers on Linux, BSD, macOS and Windows.
 
-### Alpine Linux
-
-```sh
-apk add alacritty
-```
-
-### Arch Linux
-
-```sh
-pacman -S alacritty
-```
-
-### Fedora
-
-Unofficial builds of stable tags can be found in Fedora Copr:
-[pschyska/alacritty](https://copr.fedorainfracloud.org/coprs/pschyska/alacritty/).
-
-``` sh
-dnf copr enable pschyska/alacritty
-dnf install alacritty
-```
```

Comment:
We should likely keep those somewhere in INSTALL.md, since this is a know third party repo for fedora. And so, you can't find it on your own with package manager.